### PR TITLE
mohammad/mt5

### PR DESCRIPTION
**Pull request by binary-hamedanchi**
Add MT5 pages

### CODE REVIEW

File: src/templates/metatrader/start.html.tt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<h1>[% l("Start Trading with MetaTrader 5") %]</h1>
+<div class="gr-row">
+    <div class="gr-6 gr-12-m">
+        <ol>
+            <li>[% l('Click the bottun below to download MetaTrader') %]</li>
+            <li>[% l('Run the installation file and follow the instructions') %]</li>
+            <li>[% l('To log into the application, use your MtaTrader account number and password') %]</li>
```
Comment (jy-binary) [inline src/templates/metatrader/start.html.tt]: typo in MtaTrader

Comment (binary-hamedanchi) [inline src/templates/metatrader/start.html.tt]: Fixed. Thanks.

File: src/templates/metatrader/start.html.tt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<h1>[% l("Start Trading with MetaTrader 5") %]</h1>
+<div class="gr-row">
+    <div class="gr-6 gr-12-m">
+        <ol>
+            <li>[% l('Click the bottun below to download MetaTrader') %]</li>
```
Comment (jy-binary) [inline src/templates/metatrader/start.html.tt]: typo in 'button'

Comment (binary-hamedanchi) [inline src/templates/metatrader/start.html.tt]: Fixed. Thank you.

**Pull request merged**